pkgs/stdenv/linux/make-bootstrap-tools-cross.nix

```diff
@@ -14,7 +14,7 @@ let
       float = "soft";
       withTLS = true;
       libc = "glibc";
-      platform = lib.systems.platforms.sheevaplug;
+      platform = pkgsNoParams.platforms.sheevaplug;
```

Comment:
Didn't you just merge a PR to master that did the opposite change (pkgsNoParame -> lib.systems)?